pages/index.vue

```diff
@@ -64,122 +63,154 @@
-              alt="Charles F. Vatterott, Vatterott Foundation Co-Founder"
-            />
-            <figcaption>Charles F. Vatterott</figcaption>
-          </figure>
-        </div>
         <p class="home-text">
-          During a time in which anti-racist dissent often resulted in perilous consequences, the Vatterott Foundation worked to preserve the dignity of all people. Since its first donation to the
-          <a
-            href="https://books.google.com/books?id=z5V9t5b79TwC&pg=PA140&lpg=PA140&dq=Catholic+Interracial+Council+of+St.+Louis&source=bl&ots=PErB57QUF5&sig=6B76hdc4df0Qa3KhAyNVLbPYPBU&hl=en&sa=X&ved=0ahUKEwjwxsbYrJ_QAhVTImMKHQN-CK8Q6AEIPTAH#v=onepage&q=Catholic%20Interracial%20Council%20of%20St.%20Louis&f=false"
-            target="_blank"
-          >Catholic Interracial Council</a> on October 27, 1948, the Vatterott Foundation remains a committed advocate in promoting racial justice.
+          {{ history.firsthistoryquote }}
         </p>
-        <blockquote class="big-quote">
-          <p>
-            Our home, while never destitute, was
-            <strong>poor but always happy</strong>. Our parents insisted on us unselfishness and the blessings of poverty; dignity of men; the nobility of labor and&mdash;in full bloom&mdash;sense of responsibility.
+        <div class="picture-display">
+          <div
+            v-for="figure in history.figures"
+            :key="figure.image"
+            class="figures"
+          >
+            <figure>
+              <img :src="figure.image" />
+              <figcaption>{{ figure.caption }}</figcaption>
+            </figure>
+          </div>
+        </div>
+        <div>
+          <p class="home-text">
+            {{ history.secondhistoryquoteA }}
+            <a
+              href="https://books.google.com/books?id=z5V9t5b79TwC&pg=PA140&lpg=PA140&dq=Catholic+Interracial+Council+of+St.+Louis&source=bl&ots=PErB57QUF5&sig=6B76hdc4df0Qa3KhAyNVLbPYPBU&hl=en&sa=X&ved=0ahUKEwjwxsbYrJ_QAhVTImMKHQN-CK8Q6AEIPTAH#v=onepage&q=Catholic%20Interracial%20Council%20of%20St.%20Louis&f=false"
+              target="_blank"
+              >{{ history.quotelinkA }}</a
+            >
+            {{ history.secondhistoryquoteB }}
           </p>
+        </div>
+        <blockquote class="big-quote">
+          <blockquote v-html="history.secondhistoryblockquote" />
           <footer>
-            <cite class="author">Charles F. Vatterott</cite>
-            <cite class="citation-title">Vatterott Foundation Co-Founder</cite>
+            <cite class="author">{{ history.blockquoteauthor }}</cite>
+            <cite class="citation-title">{{
+              history.blockquoteauthortitle
+            }}</cite>
           </footer>
         </blockquote>
         <p class="home-text">
-          For many years the Foundation supported the
+          {{ history.thirdhistoryquoteA }}
           <a
             href="http://archstl.org/archives/page/holy-angelsour-lady-angels-kinloch"
```

Comment:
Let's add this link as well to markdown to authors can edit it

Reply:
done-ski